api/envoy/admin/v3/config_dump.proto

```diff
@@ -58,9 +81,13 @@ message UpdateFailureState {
   // Details about the last failed update attempt.
   string details = 3;
 
-  // This is the version of the rejected resource.
+  // This is the version of the latest update for this resource.
   // [#not-implemented-hide:]
   string version_info = 4;
+
+  // Error status code of this config.
+  // [#not-implemented-hide:]
+  ConfigErrorStatus status = 5;
```

Comment:
Is this field going to be used on xDS servers?  If not, suggest calling it `client_status`.

Reply:
Name updated.